src/client/components/UserHeader.less

```diff
@@ -68,6 +69,10 @@
   }
 }
 
+.UserHeader--follows-you-indicator {
+  margin-top: -30px;
```

Comment:
You really should not be using this... to position elements, this gives the element almost a fixed position

Reply:
Probably why it's breaking on my screen

Reply:
Helo @jm90m. Can you advice on how to position? i'm no css expeet and need to learn.

Reply:
look up css flex box, lots of resources online for responsive layouts.

Reply:
ok